### PR TITLE
Clean jump notebook

### PR DESCRIPTION
**Pull request by safiyecelik**
# What?

-  I am simplifying the jump notebook as it was originally built for the cytodata workshop and has things that do not necessarily need to be here in this repo. I am also adding the efaar pipeline functions to efaar.py script for modularity.

# Why?

-   because we want an easier life for users who want to run the notebook.

### CODE REVIEW

File: efaar_benchmarking/data_loading.py [hunk 1]
```diff
@@ -1,17 +1,98 @@
...
+
+def load_cpg16_crispr(data_path="data/", filter_by_intensity=True, filter_by_cell_count=True, drop_image_cols=True):
+    plate_file_name = "plate.csv.gz"
+    well_file_name = "well.csv.gz"
+    crispr_file_name = "crispr.csv.gz"
+    plate_file_path = os.path.join(data_path, plate_file_name)
+    well_file_path = os.path.join(data_path, well_file_name)
+    crispr_file_path = os.path.join(data_path, crispr_file_name)
+    if not (os.path.exists(plate_file_path) and os.path.exists(well_file_path) and os.path.exists(crispr_file_path)):
+        source_path = "https://zenodo.org/records/7661296/files/jump-cellpainting/datasets-v0.5.0.zip?download=1"
```
Comment (ctilling) [inline efaar_benchmarking/data_loading.py]: where does this come from?

File: efaar_benchmarking/data_loading.py [hunk 2]
```diff
@@ -1,17 +1,98 @@
...
+                executer.submit(
+                    load_plate_features, CP_FEATURE_FORMATTER.format(**row.to_dict())
+                ): CP_FEATURE_FORMATTER.format(**row.to_dict())
+                for _, row in cripsr_plates.iterrows()
+            }
+            for future in as_completed(future_to_plate):
+                features.append(future.result())
+        pd.concat(features).to_parquet(features_file_path)
+    features = pd.read_parquet(features_file_path).dropna(axis=1)
+    if filter_by_intensity:
```
Comment (ctilling) [inline efaar_benchmarking/data_loading.py]: I think it might be worth splitting out the loading from the processing

**Review by ctilling — COMMENTED** [review]
Some quick thoughts on a first glance

File: efaar_benchmarking/data_loading.py (hunk 1 above)
Comment (safiyecelik) [reply efaar_benchmarking/data_loading.py]: I believe this is where you downloaded the three csv files you put in the repo, no? I want to have everything downloaded from scratch.

File: efaar_benchmarking/data_loading.py (hunk 2 above)
Comment (safiyecelik) [reply efaar_benchmarking/data_loading.py]: Done in the new commit.

**Review by ctilling — DISMISSED** [review]
Looks good to me, thanks for cleaning up the notebook!

**Pull request merged**